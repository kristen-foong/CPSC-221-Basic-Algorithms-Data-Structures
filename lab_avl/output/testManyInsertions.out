~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[1mTesting Many Insertions[22m
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
rotateRight
rotateRight
rotateRight
rotateRight
rotateRight
rotateLeft
rotateRight
rotateRight
rotateRight
rotateLeft
rotateRight
                                                                                                ______________________________ 94 _____________________________                                                                                                  
                                                                 ______________________________/                                                               \______________________________                                                                   
                                                ______________ 87 _____________                                                                                                 ______________ 96                                                                
                                 ______________/                               \______________                                                                   ______________/                                                                                 
                        ______ 42 _____                                                        92                                                              95                                                                                                
                 ______/               \______                                                                                                                                                                                                                   
            __ 17 _                         __ 78 _                                                                                                                                                                                                              
         __/       \__                   __/       \__                                                                                                                                                                                                           
       3               41              45              82                                                                                                                                                                                                        
                     /                   \                                                                                                                                                                                                                       
                   36                      73                                                                                                                                                                                                                    
                                          /                                                                                                                                                                                                                      
                                         71                                                                                                                                                                                                                      


