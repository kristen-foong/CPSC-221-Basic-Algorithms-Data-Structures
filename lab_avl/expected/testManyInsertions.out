~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[1mTesting Many Insertions[22m
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
rotateRight
rotateRight
rotateRight
rotateRight
rotateRightLeft
rotateRight
rotateLeft
rotateLeft
rotateLeftRight
rotateLeft
rotateRight
rotateRight
                        ______ 61 _____                          
                 ______/               \______                   
            __ 26 _                         __ 87 _              
         __/       \__                   __/       \__           
       17              36              76              94        
     /   \           /   \           /   \           /   \       
   3       23      31      42      73      78      92      96    
  / \     / \             / \     / \       \       \     /      
 1   11  18  24          41  45  71  74      82      93  95      


